sorbet/rbi/shims/uri.rbi

```diff
@@ -0,0 +1,14 @@
+# typed: strict
+# frozen_string_literal: true
+
+module URI
```

Comment:
Possible.

Looking at the `sorbet/tapioca/require.rb`, we do require `ruby-lsp/internal` which require the core ext 🤔 

Let's keep the shim then 👍

Reply:
Yeah the reason this method isn't being generated by Tapioca is because calling `source_location` on the method object returns `[....gem/ruby/3.2.1/gems/sorbet-runtime-0.5.10950/lib/types/private/methods/_methods.rb", 255]`. [This is the the line that's referenced in sorbet-runtime](https://github.com/sorbet/sorbet/blob/master/gems/sorbet-runtime/lib/types/private/methods/_methods.rb#L255C6-L255C6). I'm not sure why this is but I could open a ticket on Tapioca if that's helpful?